Sources/Sentry/include/SentrySwizzle.h

```diff
@@ -347,33 +351,82 @@ typedef NS_ENUM(NSUInteger, SentrySwizzleMode) {
 // and remove it later.
 #define _SentrySWArguments(arguments...) DEL, ##arguments
 
-#define _SentrySwizzleInstanceMethod(classToSwizzle, selector, SentrySWReturnType,                 \
-    SentrySWArguments, SentrySWReplacement, SentrySwizzleMode, KEY)                                \
-    [SentrySwizzle                                                                                 \
-        swizzleInstanceMethod:selector                                                             \
-                      inClass:[classToSwizzle class]                                               \
-                newImpFactory:^id(SentrySwizzleInfo *swizzleInfo) {                                \
-                    SentrySWReturnType (*originalImplementation_)(                                 \
-                        _SentrySWDel3Arg(__unsafe_unretained id, SEL, SentrySWArguments));         \
-                    SEL selector_ = selector;                                                      \
-                    return ^SentrySWReturnType(_SentrySWDel2Arg(                                   \
-                        __unsafe_unretained id self, SentrySWArguments)) { SentrySWReplacement };  \
-                }                                                                                  \
-                         mode:SentrySwizzleMode                                                    \
-                          key:KEY];
-
-#define _SentrySwizzleClassMethod(                                                                 \
-    classToSwizzle, selector, SentrySWReturnType, SentrySWArguments, SentrySWReplacement)          \
-    [SentrySwizzle                                                                                 \
-        swizzleClassMethod:selector                                                                \
-                   inClass:[classToSwizzle class]                                                  \
-             newImpFactory:^id(SentrySwizzleInfo *swizzleInfo) {                                   \
-                 SentrySWReturnType (*originalImplementation_)(                                    \
-                     _SentrySWDel3Arg(__unsafe_unretained id, SEL, SentrySWArguments));            \
-                 SEL selector_ = selector;                                                         \
-                 return ^SentrySWReturnType(_SentrySWDel2Arg(                                      \
-                     __unsafe_unretained id self, SentrySWArguments)) { SentrySWReplacement };     \
-             }];
+#if TEST
```

Comment:
`h`: I think it would be acceptable to remove the test build configuration. If we don't call the original method, the DSK should crash in every build configuration.